nncf/torch/nncf_network.py

```diff
@@ -716,13 +718,15 @@ def _collect_eval_op_scopes(self, model: nn.Module, dummy_forward_fn: Callable)
         return result
 
     def get_node_to_op_address_mapping(self) -> Dict[NNCFNodeName, OperationAddress]:
-        # The IDs of corresponding nodes of the original dynamic graph and original NNCF graph
-        # must be equal for this to work.
         retval = {}
-        for node in self._original_dynamic_graph.get_all_nodes():
+        dynamic_graph = (
+            self._original_dynamic_graph if self._compressed_traced_graph is None else self._compressed_traced_graph
+        )
+        nncf_graph = self._original_graph if self._compressed_graph is None else self._compressed_graph
```

Comment:
is it possible that `self._compressed_traced_graph` is exists but `self._compressed_graph`? 


```python
if self._compressed_graph is None:
   dynamic_graph = self._original_dynamic_graph
   nncf_graph = self._original_graph
else: 
   dynamic_graph = self._compressed_traced_graph
   nncf_graph = self._compressed_graph
```

Reply:
It is not possible in current code, but I would prefer to check both graphs explicitly as dynamic graph and NNCFGraph are not explicitly depend on each other. I would rather avoid introducing of this implicit dependency

Reply:
I made a `PTGraphPair` to make it clear, please check